components/services/bat_ledger/bat_ledger_impl.cc

```diff
@@ -392,4 +392,22 @@ void BatLedgerImpl::GetRewardsInternalsInfo(
   std::move(callback).Run(info.ToJson());
 }
 
-}  // namespace bat_ledger
+void BatLedgerImpl::GetContributionIsInProgress(
+      GetContributionIsInProgressCallback callback) {
+  auto* holder = new CallbackHolder<GetContributionIsInProgressCallback>(
+      AsWeakPtr(), std::move(callback));
+
+  ledger_->GetContributionIsInProgress(
+      std::bind(BatLedgerImpl::OnGetContributionIsInProgress, holder, _1));
+}
+
+void BatLedgerImpl::OnGetContributionIsInProgress(
```

Comment:
nit: this one should be before `GetContributionIsInProgress` as it's used in `GetContributionIsInProgress`

Reply:
Fixed